src/ORM/TableRegistry.php

```diff
@@ -55,6 +55,9 @@
  * // Prior to 3.6.0
  * $table = TableRegistry::get('Users', $config);
  * ```
+ *
+ * @deprecated 4.1.0 Use \Cake\ORM\Locator\LocatorAwareTrait::getTableLocator()
```

Comment:
Can we add a {@link or see tag here too, to be able to follow the link?

Reply:
The second example has an argument too, will IDEs be able to navigate `@see`/`@link` tag for that?

Reply:
We might not need to, the first is usually useful enough.